solutionbox/inception/datalab_solutions/inception/_cloud.py

```diff
@@ -67,55 +64,41 @@ def preprocess(self, input_csvs, labels_file, output_dir, pipeline_option=None):
         p, self._checkpoint, input_csvs, labels_file, output_dir, job_name)
     p.run()
 
-  def train(self, labels_file, input_dir, batch_size, max_steps, output_path, credentials,
+  def train(self, labels_file, input_dir, batch_size, max_steps, output_path,
             region, scale_tier):
     """Cloud training with CloudML trainer service."""
 
+    import datalab.mlalpha as mlalpha
     num_classes = len(_util.get_labels(labels_file))
-    job_id = 'inception_train_' + datetime.datetime.now().strftime('%y%m%d_%H%M%S')
-    job_args_dict = {
+    job_args = {
```

Comment:
side question: is there a reason to prefer making a cloud_runner.run call vs a subprocess call to gcloud?